### PR TITLE
Eyepatch fix + new eyepatch

### PR DESCRIPTION
**Pull request by Greenjoe12345**
This PR adjusts the sprites on the eyepatch so it doesn't cover both eyes, it also adds a new, recolorable eyepatch, available in the loadout.

### CODE REVIEW

**Comment by Greenjoe12345** [comment]
!review

File: code/modules/clothing/glasses/glasses.dm [hunk 1]
```diff
@@ -277,6 +277,12 @@ BLIND     // can't see anything
 	update_clothing_icon()
 	update_icon()
 
+/obj/item/clothing/glasses/eyepatch/white
+	name = "eyepatch"
+	desc = "A simple eyepatch made of a strip of cloth"
```
Comment (alsoandanswer) [inline code/modules/clothing/glasses/glasses.dm:282]:
```suggestion
	desc = "A simple eyepatch made out of a strip of cloth."
```

**Comment by alsoandanswer** [comment]
to make a changelog, copy and paste example.yml in the html/changelogs folder in your aurora folder.

then, edit example.yml accordingly, and push

**Comment by Greenjoe12345** [comment]
oop forgot about that

**Review by alsoandanswer — APPROVED** [review]

**Review by Geevies — APPROVED** [review]

**Pull request merged**